src/CRUD.php

```diff
@@ -165,16 +165,7 @@ public function initCreate()
      */
     public function jsSaveCreate()
     {
-        return [
-            // reload Grid
-            (new jQuery($this))->trigger('reload'),
-
-            // close modal
-            new jsExpression('$(".atk-dialog-content").trigger("close")'),
-
-            // display notification
-            $this->factory($this->notifyCreate ?: $this->notifyDefault),
```

Comment:
You are loosing notifyCreate message now.

Reply:
should be ok now!